packages/svelteui-core/src/components/Chip/Chip.stories.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+  import { Meta, Template, Story } from '@storybook/addon-svelte-csf';
+  import { Chip } from './index';
+  import { Group } from '../Group';
+  import {Stack} from "../Stack";
+  import { useSvelteUITheme } from '$lib/styles';
+
+  const theme = useSvelteUITheme();
+  const colors = Object.keys(theme.colorNames);
+
+</script>
+
+<Meta title="Components/Chip" component={Chip} />
+
+<Template let:args>
+  <Chip {...args} ></Chip>
+</Template>
+
+<Story name="Chip" />
+
+<Story name="Colors">
+  <div class="{theme.colorScheme}">
+  <Stack>
+    <Group>
+      <Chip>Default chip</Chip>
+      <Chip color="green" variant="filled">Filled chip</Chip>
+      <Chip color="red" variant="outline" checked>Outline chip</Chip>
+      <Chip disabled>Disabled chip</Chip>
+      <Chip checked disabled>Checked disabled chip</Chip>
+      <Chip size="xl">Large chip</Chip>
```

Comment:
This could kinda be extracted to a new story called `States`, but this is just nitpicking